llvm/test/MC/AsmParser/labels.s

```diff
@@ -46,8 +46,8 @@ foo:
 
 // FIXME: We don't bother to support .lsym.
 
-// COM: CHECK: .lsym "a 8",1
-//        .lsym "a 8", 1
+// CHECK: .lsym "a 8",1
+.lsym "a 8", 1
```

Comment:
C:\ws\src\llvm\test\MC\AsmParser\labels.s:51:1: error: directive '.lsym' is unsupported